ssiva13/Questioner
# Ft admin delete meetup 162978053

**What Does This PR Do?**
This PR contains the UI template for the delete meetup feature.
**Description of Feature/Task**
Create delete meetup feature page

**_How to Manually test this feature?_**

- Clone the repository using 
- cd into project folder
- Open delete.html using a browser


> Pivotal tracker story ID _#162978053_

The page(as it should be)

![screenshot_2019-01-06 questioner 7](https://user-images.githubusercontent.com/20058478/50740667-e011ac80-1202-11e9-842f-cda6b44c00df.png)
